package.json

```diff
@@ -11,12 +11,14 @@
     "build:production": "NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js"
   },
   "dependencies": {
-    "bootstrap": "^3.3.7",
+    "bootstrap": "^4.1.3",
     "d3": "^5.5.0",
-    "laravel-mix": "^2.1.11"
+    "laravel-mix": "^2.1.11",
+    "popper.js": "^1.14.5"
   },
   "devDependencies": {
     "browser-sync": "^2.24.4",
-    "browser-sync-webpack-plugin": "^2.2.2"
+    "browser-sync-webpack-plugin": "^2.2.2",
+    "jquery": "1.12.4"
```

Comment:
when I remove `"jquery": "1.12.4"`, I don't see any change in behaviour. Can you remind me again, why this is here? Would prefer to have it removed

Reply:
When I don't include it I get the following error message on yarn build

ERROR in ./node_modules/bootstrap/dist/js/bootstrap.bundle.min.js
Module not found: Error: Can't resolve 'jquery' in '/Users/brettf/Documents/Freelance work/New work/bccampus/pd.test/site/web/app/themes/eypd/node_modules/bootstrap/dist/js'
 @ ./node_modules/bootstrap/dist/js/bootstrap.bundle.min.js 6:78-95
 @ multi ./node_modules/bootstrap/dist/js/bootstrap.bundle.min.js ./assets/styles/main.scss ./assets/styles/login.scss ./assets/styles/admin.scss ./assets/styles/event.scss ./assets/styles/media.scss ./assets/styles/tinymce.scss
error Command failed with exit code 2.

Reply:
my understanding is that we would want to see it in the dependencies list, rather than the devDependencies:

![image](https://user-images.githubusercontent.com/2048170/48654084-dd42ca00-e9be-11e8-861b-2bbc61cd7685.png)

Reply:
I started there but it was throwing other errors. Please correct me if I am wrong but if its in the dependencies doesn't it get baked into the code where devdependencies are just for dev. In this case my reasoning is jquery is already in WP so why are we including it as an actual dependency? BS4 also support 1.12.4. Including a version higher then then seems like asking for problems. Again if I am misunderstanding how this works please let me know.